src/application/components/Explorer/postMessageHelpers.ts

```diff
@@ -20,17 +21,30 @@ export const EMBEDDABLE_EXPLORER_URL =
 export const EXPLORER_SUBSCRIPTION_TERMINATION =
   "ExplorerSubscriptionTermination";
 
-export type ExplorerResponse = {
```

Comment:
This type was out of date so I updated it to be in line with our embeddable explorer types.